bigquery/job.go

```diff
@@ -398,14 +421,19 @@ type JobStatistics struct {
 	// a script.
 	ScriptStatistics *ScriptStatistics
 
-	// ReservationUsage attributes slot consumption to reservations.
-	ReservationUsage []*ReservationUsage
-
```

Comment:
ops, better to leave this field and update the comment about deprecation, because that would be a breaking change it we remove it right away.

Reply:
ah, was just reordered, sorry